modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1809,10 +1806,16 @@ sub _munge_meta {
       $dedupe->{$v}++;
     }
 
-    $self->tree->{'SAMPLE_DATA'} = $sample_hash if scalar keys %$sample_hash;
+    $self->tree($production_name)->{'SAMPLE_DATA'} = $sample_hash if scalar keys %$sample_hash;
 
     # check if the karyotype/list of toplevel regions ( normally chroosomes) is defined in meta table
-    @{$self->tree->{'TOPLEVEL_REGIONS'}} = @{$meta_hash->{'regions.toplevel'}} if $meta_hash->{'regions.toplevel'};
+    @{$self->tree($production_name)->{'TOPLEVEL_REGIONS'}} = @{$meta_hash->{'regions.toplevel'}} if $meta_hash->{'regions.toplevel'};
+
+    ## need to explicitly define as empty array by default
+    ## SpeciesDefs looks for a value at collection level
+    if ($self->is_collection('DATABASE_CORE')) {
+      @{$self->tree($production_name)->{'ENSEMBL_CHROMOSOMES'}} = ();                                     @{$self->tree($production_name)->{'ENSEMBL_CHROMOSOMES'}} = @{$meta_hash->{'region.toplevel'}} if $meta_hash->{'region.toplevel'};
```

Comment:
Not a big thing but the styling is bothering me, why the @{$self...} after the semi colon going on the same line, it should be another line.

Reply:
How odd! It looked fine in vim, but there were spaces instead of a line break. I suspect the same formatting exists in eg-web-common - I recall seeing odd indenting in lots of places bc Eugene K didn't set up his .vimrc correctly.